Assets/Scripts/HUD/ProgressBar.cs

```diff
@@ -4,9 +4,13 @@
 using UnityEngine.UI;
 public class ProgressBar : MonoBehaviour
 {
+
+    [SerializeField] private Text _SliderText;
```

Comment:
For fields, please make it so it's like _slideText instead
And for methods, please make it's so it a captial letter first

Reply:
Sorry :D im still used to java. I will implement the feedback and recommit